src/sage/misc/viewer.py

```diff
@@ -301,6 +301,7 @@ def __call__(self, x=None):
         elif x.startswith('pdf'):
             return self.pdf_viewer()
 
+
 viewer = Viewer()
 
 def browser():
```

Comment:
No two blank line below L305? 

There is only one blank line before the class definition.